src/mscorlib/corefx/Interop/Windows/BCrypt/Interop.BCryptGenRandom.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
This one can be shared, I had it unshared because the equivalent Unix implementation was different. I will move this one.